src/components/comment/comment.scss

```diff
@@ -2,9 +2,24 @@
 @use '../../compiled/tokens/scss/color';
 @use '../../compiled/tokens/scss/font-weight';
 @use '../../compiled/tokens/scss/size';
+@use '../../mixins/emdash';
 @use '../../mixins/headings';
 @use '../../mixins/theme';
 
+/**
+ * Themed properties
+ */
+
+@include theme.props() {
+  --theme-color-border-blockquote: #{color.$border-blockquote-light};
+  --theme-color-text-blockquote: #{color.$text-blockquote-light};
+}
+
+@include theme.props(dark) {
+  --theme-color-border-blockquote: #{color.$border-blockquote-dark};
+  --theme-color-text-blockquote: #{color.$text-blockquote-dark};
+}
```

Comment:
Reading this now, I think for the text you might be able to use `--theme-color-text-muted` instead of defining new properties and tokens? See `src/base/_themes.scss`

I could see adding a new border property in the base themes as well, esp. since we're wanting to coordinate that color with comment threads.

Reply:
Cool! I didn't know we had theme tokens. I updated the color and border color properties so that we're using existing tokens.